planet/subscription_request.py

```diff
@@ -212,6 +219,86 @@ def catalog_source(
     return {"type": "catalog", "parameters": parameters}
 
 
+def planetary_variable_source(
+    var_type: Literal["biomass_proxy",
+                      "land_surface_temperature",
+                      "soil_water_content",
+                      "vegetation_optical_depth"],
+    var_id: str,
+    geometry: Mapping,
+    start_time: datetime,
+    end_time: Optional[datetime] = None,
+) -> dict:
+    """Construct a Planetary Variable subscription source.
+
+    Planetary Variables come in 4 types and are further subdivided
+    within these types. See [Subscribing to Planetary
+    Variables](https://developers.planet.com/docs/subscriptions/pvs-subs/#planetary-variables-types-and-ids)
+    for details.
+
+    The return value can be passed to
+    [planet.subscription_request.build_request][].
+
+    Note: this function does not validate variable types and ids.
+
+    Parameters:
+        var_type: one of "biomass_proxy", "land_surface_temperature",
+            "soil_water_content", or "vegetation_optical_depth".
+        var_id: a value such as "SWC-AMSR2-C_V1.0_100" for soil water
+            content derived from AMSR2 C band.
```

Comment:
Would you be opposed to pointing a user to all of the possible `var_id`s? I.e., 
```suggestion
        var_id: a value such as "SWC-AMSR2-C_V1.0_100" for soil water
            content derived from AMSR2 C band. See all options in the [Planetary Variables docs](https://developers.planet.com/docs/subscriptions/pvs-subs/#create-a-planetary-variables-subscription).
```

Reply:
@kevinlacaille you mean link to https://developers.planet.com/docs/subscriptions/pvs-subs/#planetary-variables-types-and-ids instead? Or something else?

Reply:
Oh yes, that's what I meant to send. I suppose this is optional, but especially for non-intuitive names, this would be helpful. Thoughts?